### DISCUSSION TITLE
If Alignment false, don't send data forward

### DISCUSSION THREAD

**Discussion by JanneN9** [jomjol/AI-on-the-edge-device#1920] [Ideas] [answered]
Hei,

Reason for this is sequence of Power meter LCD
 - show date 10s, show time 10s, show power usage 10s

If Power Usage is on LCD there is "nice" alignment text in upper corner: "kWh".
![image](https://user-images.githubusercontent.com/10402422/215066102-049ed45b-78cc-4003-b128-5944cb5c70aa.png)

Idea:
 - if alignment fail
   - wait 5s
   - capture new image and do_flow()
   (- loop 2 times (if first one were date))
   (- loop 1 times (if first one were time)) 

Currently looking code how to implement this. But code is not C or C# or Cuda, so it takes time to find correct location for this.

Or it is implemented with
 - ErrorMessage = false ?!?

Terveisin
Janne

**Comment by caco3** [comment, marked as answer]
You are not the first one requesting such thing, so I created a feature request for this: https://github.com/jomjol/AI-on-the-edge-device/issues/1936
